packages/amplify-category-api/src/provider-utils/awscloudformation/container-artifacts.js

```diff
@@ -0,0 +1,189 @@
+export const containerFiles = {
```

Comment:
What's the difference between container-artifacts.js  vs container-artifacts-graphql.js?

Reply:
@attilah What do you recommend here when you say best practice? Also what do you mean a "REST one" with a single endpoint?